ui/src/Model.elm

```diff
@@ -33,51 +32,67 @@ init location =
         updatedModel =
             routeFromUrl defaultModel location
     in
-        updatedModel ! [ getBuildRecordList updatedModel ]
+        updatedModel
+            ! [ getFilters "product"
+              , getFilters "channel"
+              , getFilters "platform"
+              , getFilters "version"
+              , getFilters "locale"
+              , getBuildRecordList updatedModel
+              ]
+
+
+getFilters : String -> Cmd Msg
+getFilters filterName =
+    client
+        |> Kinto.getList (filterRecordResource filterName)
+        |> Kinto.sortBy [ "name" ]
```

Comment:
I would rather sort by ``id`` I am working on a patch to make sure the version are ordered in the right order (not the string one but really the version one.)